### DISCUSSION TITLE
how to create floating stacked bar chart

### DISCUSSION THREAD

**Discussion by gane27** [Abhinandan-Kushwaha/react-native-gifted-charts#407] [Q&A] [answered]
How to create this below floating bar chart using stacked bar chart using gifted chart

![WhatsApp Image 2023-10-12 at 7 32 37 PM](https://github.com/Abhinandan-Kushwaha/react-native-gifted-charts/assets/33153210/1e6c1bda-2c13-431f-bf68-96ff79bc678a)

**Comment by Abhinandan-Kushwaha** [comment, marked as answer]
Hey @gane27 👋
Sorry for the late reply. Try adding `barMarginBottom` to the objects in the `stackData` array. Let me know if it doesn't help.
